tools/rosbash/scripts/rosrun

```diff
@@ -48,12 +48,14 @@ case $2 in
 esac
 
 if [[ -n $CMAKE_PREFIX_PATH ]]; then
-  catkin_package_libexec_dir=`catkin_find --without-underlays --libexec --share $1 2> /dev/null`
-  debug "Looking in catkin libexec dir: $catkin_package_libexec_dir"
+  IFS=$'\n'
```

Comment:
`$'\n'` is the easiest way to get a literal newline in bash. You can try it yourself:
```
$ foo='\n'
$ bar="\n"
$ echo $foo$bar
```
prints: `\n\n`